### PR TITLE
make cli BIDS-Apps compliant

### PR DESCRIPTION
**Pull request by shoshber**
https://github.com/poldracklab/preprocessing-workflow/issues/106

### CODE REVIEW

**Comment by oesteban** [comment]
can you rebase? thx!

**Comment by shoshber** [comment]
merged

File: fmriprep/run_workflow.py [hunk 1]
```diff
@@ -28,14 +30,24 @@ def main():
     from nipype.pipeline import engine as pe
     from fmriprep import __version__
     from fmriprep.workflows import fmriprep_single
-
+    from fmriprep.utils.misc import collect_bids_data
```
Comment (oesteban) [inline fmriprep/run_workflow.py]: import should go away

File: fmriprep/run_workflow.py [hunk 2]
```diff
@@ -124,10 +131,13 @@ def main():
 
     # Determine subjects to be processed
     subject_list = opts.subject_id
-    if not subject_list or subject_list is None:
+
+    if not subject_list or len(subject_list) == 0:
         subject_list = [op.basename(subdir)[4:] for subdir in glob.glob(
             op.join(settings['bids_root'], 'sub-*'))]
 
+    logger.info("subject list: " + pprint.pformat(subject_list))
```
Comment (oesteban) [inline fmriprep/run_workflow.py]: subject_list is just a list, so we can also remove the pprint: ```logger.info("subject list: {}", ', '.join(subject_list))```

**Comment by oesteban** [comment]
I'm still not convinced by changing the collect_bids_data return object because the tests expect it to be in a certain way. I would prefer to adapt the tests, regardless we eventually move to a result object more similar to what you expect now in the tests.

**Comment by shoshber** [comment]
Yeah I forgot this pr had those commits in it too. I will cherrypick and make a new pr for this as well.

**Pull request closed without merging**